backend/location/models.py

```diff
@@ -67,6 +67,14 @@ class Location(models.Model):
         null=True,
     )
 
+    osm_type = models.CharField(
```

Comment:
The Django enumeration type could be an option here for this new field.
https://docs.djangoproject.com/en/3.0/ref/models/fields/#enumeration-types